src/test/java/uk/gov/hmcts/cmc/claimstore/events/offer/AgreementCountersignedStaffNotificationHandlerTest.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.cmc.claimstore.events.offer;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.cmc.claimstore.services.staff.SettlementReachedStaffNotificationService;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.offers.MadeBy;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.offers.SampleSettlement;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+public class AgreementCountersignedStaffNotificationHandlerTest {
+
+    @Mock
+    private SettlementReachedStaffNotificationService settlementReachedStaffNotificationService;
+    private AgreementCountersignedStaffNotificationHandler handler;
+    private AgreementCountersignedEvent event;
+
+    @BeforeEach
+    public void setup() {
+        event = new AgreementCountersignedEvent(SampleClaim.getWithSettlement(SampleSettlement.validDefaults()),
+            MadeBy.CLAIMANT,
+            "authorisation");
+    }
+
+    @Nested
+    @DisplayName("Staff email notification feature toggle test")
+    class StaffEmailNotificationFeatureToggle {
```

Comment:
No need to nest these. If you just want the nice display name, you can put the annotation on the test methods themselves